apps/web/src/features/layout/components/Starfield/starfield.module.css

```
@@ -456,23 +563,43 @@
 @keyframes fadeIn {
-  from { opacity: 0; }
-  to { opacity: 1; }
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
 }
 
 @keyframes fadeOut {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)